apps/src/templates/progress/ProgressButton.jsx

```diff
@@ -0,0 +1,136 @@
+/**
+ * New style button. Eventually we'd like to have this used more broadly (at which
+ * point it should be renamed to something more generic)
+ * This particular button is designed to operate in contexts where we have a solid
+ * background. When we're a button on top of an image, we may want something different.
+ */
```

Comment:
Wasn't too sure what I should call this. Intention is that this become the button used everywhere, not just in Progress but (a) that's not the case so far and (b) we already have a component elsewhere called just Button, and I wasn't too sure how to differentiate.

Probably/possibly this should be named something else and live somewhere else?